### ISSUE TITLE
best loss of 0 in hyper band search

### ISSUE THREAD

**Issue by TimeDelta** [TimeDelta/q-vs-c_enc_dec_nonstationary#3]
print(f"  Round {round_index}: {epochs_this_round} epochs; best loss = {min(round_losses):.4f}") gave "best loss = 0.0000" for in round 0 (single epoch per config)
